week4/notes/http-notes.md

```diff
@@ -55,28 +56,31 @@
   - HEAD
   - OPTIONS
   - TRACE
-  
+  - CONNECT
```

Comment:
I only added missing CONNECT http method. Everything else is vscode mardown extension formatting on save.

Thanks.